frameworks/rooch-framework/sources/transfer.move

```diff
@@ -19,10 +22,7 @@ module rooch_framework::transfer {
         to: address,
         amount: u256,
     ) {
-        if(!account::exists_at(to)) {
-            account_entry::create_account(to);
-        };
-
+        assert!(account::exists_at(to), ErrorAccountNotExists);
```

Comment:
We should allow the transfer of coins to the no-existing account.

Reply:
Likewise, if an account is created, but no one holds the private key for that address, it is also lost.